aioresponses/compat.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-import asyncio  # noqa
+import asyncio  # noqa: F401
```

Comment:
```suggestion
import asyncio  # noqa
```
Also included in https://github.com/pnuckowski/aioresponses/pull/130 (better fit).